### PR TITLE
feat: add file qiniu cloud oss

### PR DESCRIPTION
**Pull request by stulzq**
**What this PR does**:
Add file qiniu cloud oss
**Which issue(s) this PR fixes**:

None
**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Review by wenxuwan — DISMISSED** [review]
LGTM

File: configs/config_file_qiniu_oss.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+                  "type": "grpc",
+                  "config": {
+                    "server_name": "runtime",
+                    "grpc_config": {
+                      "hellos": {
+                        "helloworld": {
+                          "hello": "greeting"
+                        }
+                      },
+                      "files": {
```
Comment (seeflood) [inline configs/config_file_qiniu_oss.json:24]: 这个配置项改成`file`了，我帮你改下吧

**Review by seeflood — APPROVED** [review]
lgtm

**Review by wenxuwan — APPROVED** [review]
LGTM

**Pull request merged**